### PR TITLE
Make submission guidance text bold

### PR DESCRIPTION
**Pull request by jamesnunn**
### What is the context of this PR?
Makes submission guidance text bold.

### How to review 
Complete test_view_submitted_response schema.json to summary page, check that the text `Please submit this survey to complete it` above the submit button is bold.

### Checklist

* [ ] New static content marked up for translation
* [ ] Newly defined schema content included in eq-translations repo

### CODE REVIEW

**Comment by jamesnunn** [comment]
[Submission guidance needs to be bold (1 SP)](https://trello.com/c/8wU93GdZ/5161-submission-guidance-needs-to-be-bold-1-sp)

File: templates/layouts/_submit.html [hunk 1]
```diff
@@ -23,7 +23,7 @@ <h1 class="ons-u-mt-m">{{ content.title }}</h1>
 
   {% if content.guidance %}
     <div class="ons-u-mt-l">
-      <p class="ons-u-mb-no" data-qa="guidance">{{ content.guidance }}</p>
+      <p class="ons-u-fs-r--b" data-qa="guidance">{{ content.guidance }}</p>
```
Comment (MebinAbraham) [inline templates/layouts/_submit.html:26]: For accessibility, we should use `<strong>` tags instead. Donna confirmed in the comments.

**Review by MebinAbraham — CHANGES_REQUESTED** [review]

File: templates/layouts/_submit.html [hunk 2]
```diff
@@ -23,7 +23,9 @@ <h1 class="ons-u-mt-m">{{ content.title }}</h1>
 
   {% if content.guidance %}
     <div class="ons-u-mt-l">
-      <p class="ons-u-mb-no" data-qa="guidance">{{ content.guidance }}</p>
+      <strong>
+        <p class="ons-u-mb-no" data-qa="guidance">{{ content.guidance }}</p>
+      </strong>
```
[the remark this answers is not in the document]
Comment (jamesnunn) [reply templates/layouts/_submit.html]: I've moved it to just around the text regardless

[the remark this answers is not in the document]
Comment (MebinAbraham) [reply templates/layouts/_submit.html]: Rich thinks wrapping strong just around the text is better, but unsure whether it has an accessibility impact. So we'll go with strong inside the p tag.

**Review by MebinAbraham — APPROVED** [review]

**Review by rmccar — APPROVED** [review]

**Pull request merged**